cristin-import/src/test/java/cucumber/ReportFeatures.java

```diff
@@ -17,14 +19,15 @@ public ReportFeatures(ScenarioContext scenarioContext) {
     }
 
     @Then("the NVA Resource Report has a PublicationContext with publisher equal to {string}")
-    public void theNvaResourceReportHasAPublicationContextWithPublisherEqualTo(String expectedPublisher) {
+    public void theNvaResourceReportHasAPublicationContextWithPublisherEqualTo(String expectedPublisherName) {
```

Comment:
Maybe we should change the scenario line to be 
```
the NVA Resource Report has a PublicationContext with a publisher with name equal to {string}
```
and add a scenario line  `that the Publisher cannot be verified througt a URI.`

Reply:
if we do that I suggest a new task for it.